test/cloudtest/pkg/commands/main.go

```diff
@@ -1056,7 +1056,17 @@ func (ctx *executionContext) findGoTest(executionConfig *config.ExecutionConfig)
 	for _, t := range execTests {
 		t.Kind = model.TestEntryKindGoTest
 		t.ExecutionConfig = executionConfig
-		result = append(result, t)
+		match := true
```

Comment:
match := len(executionConfig.OnlyRun) == 0
?

Reply:
Overall a good idea... but in the event len(executionConfig.OnlyRun) == 0... I want us to run *all* the tests.  So if we have an empty only-run list... just run everything :)